fjord/celery.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import
+
+import os
+
+from celery import Celery
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
+
+from django.conf import settings
+
+app = Celery('fjord')
+
+# Using a string here means the worker will not have to
+# pickle the object when using Windows.
+app.config_from_object('django.conf:settings')
+app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
+
+
+@app.task(bind=True)
+def debug_task(self):
+    print('Request: {0!r}'.format(self.request))
```

Comment:
Oh, it's copy paste from the first step docs. I was using it to test celery from the shell.

```
In [1]: from fjord.celery import debug_task

In [2]: debug_task.delay()
Out[2]: <AsyncResult: 1ec26e79-ca12-49c4-b30c-e6bde4c49a5d>

In [3]: exit()
```